examples/platforms/posix/uart.c

```diff
@@ -188,7 +188,7 @@ void posixUartUpdateFdSet(fd_set *aReadFdSet, fd_set *aWriteFdSet, int *aMaxFd)
 
 void posixUartProcess(void)
 {
-    const int flags = POLLRDNORM | POLLERR | POLLNVAL | POLLHUP;
+    const int flags = POLLIN | POLLRDNORM | POLLERR | POLLNVAL | POLLHUP;
```

Comment:
When I run the CLI on the Windows Subsystem for Linux, it didn't support `POLLRDNORM`, so I had to add this to get it to work.